components/motor/gpio/motor_encoder.go

```diff
@@ -354,7 +354,6 @@ func (m *EncodedMotor) directionMovingInLock() float64 {
 // SetPower sets the percentage of power the motor should employ between -1 and 1.
 // Negative power implies a backward directional rotational.
 func (m *EncodedMotor) SetPower(ctx context.Context, powerPct float64, extra map[string]interface{}) error {
-	m.opMgr.CancelRunning(ctx)
```

Comment:
If we remove this line, what happens if someone calls GoFor and then calls SetPower while it's still going?

Reply:
the power will momentarily be set to what the direct SetPower call requests, but the control loop or RPM monitor will correct the power and bring it back down or up to where it should be for the gofor call

Reply:
:grimacing: That seems like the wrong behavior: calling SetPower will in effect not set the power. In particular, I can imagine people calling `SetPower(0)` instead of `Stop()`, and expecting the motor to stop. I wonder if there's a way to keep this line and fix the problem another way.

Reply:
probably by completely redoing the way op manager handles bases and motors which is what we need to have a bigger discussion about, but imo GoFor working is more important at this moment. because if it's blocking anyways, which it should be, a SetPower call shouldn't be able to interrupt it as far as I know

Reply:
so if you did SetPower(0) in a script it wouldn't work until GoFor was done running. If you set the power to 0 in the rc card I don't know why you wouldn't just hit stop

Reply:
of course we can't assume users will have this same logic but it feels like a problem that will be handled by redoing op manager w/ motors

Reply:
We discussed in person: this PR is a temporary change to make GoFor work at all, at the cost of not being able to turn off GoFor if you want to interrupt it with something else. In the long run, we'll figure something else out, but for now this PR is way better than not having this PR. LGTM!